e2etest/newe2e_resource_manager_getter.go

```diff
@@ -24,8 +24,13 @@ func GetRootResource(a Asserter, location common.Location, varOpts ...GetResourc
 
 		return NewLocalContainer(a)
 	case common.ELocation.Blob(), common.ELocation.BlobFS(), common.ELocation.File():
-		// acct handles the dryrun case for us
-		acct := GetAccount(a, DerefOrDefault(opts.PreferredAccount, PrimaryStandardAcct))
+		// do we have a hns acct attached, if so, and we're requesting blobfs, let's use it
```

Comment:
something about this code seems like the problem with the tests. currently we are using a hns account when we shouldnt be. there are some HNS account doesnt support this yet type errors.